lib/logstash/filters/elasticsearch.rb

```diff
@@ -38,8 +38,14 @@ class LogStash::Filters::Elasticsearch < LogStash::Filters::Base
 
   # Elasticsearch query string
   config :query, :validate => :string
+ 
+  # Index to search in
+  config :index, :validate => :string, :default => ""
```

Comment:
A default value of empty string seems confusing to expose in the documentation. I'm OK with the default *behavior* being "all indices" but the documentation should just state the default behavior. Showing, in the docs, 'the default value is ""' is confusing to me.

I recommend having no `:default => ...` and just declaring the default behavior.